# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/libcap/hotfix-psx_calls.patch
# Copyright (C) 2025 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

--- libcap-2.75/psx/psx_calls.c.orig	2025-04-07 18:08:39.722333553 +0200
+++ libcap-2.75/psx/psx_calls.c	2025-04-07 18:11:13.443929723 +0200
@@ -33,9 +33,12 @@
  * want to use, we have ended up just inlining all architecture
  * support here.
  */
-#if defined(__x86_64__) || defined(__i386__)	   \
-    || defined(__arm__) || defined(__aarch64__)	   \
-    || defined(__mips__) || defined(__loongarch__) \
+
+#if defined(__x86_64__) || defined(__i386__) || defined(__ia64__) \
+    || defined(__alpha__) || defined(__arm__) || defined(__aarch64__) \
+    || defined(__hppa__) || defined(__m68k__) || defined(__nios2__) \
+    || defined(__microblaze__) || defined(__mips__) || defined(__loongarch__) \
+    || defined(__sparc__) || defined(__or1k__) || defined(__sh__) \
     || defined(__powerpc__) || defined(__s390__) || defined(__riscv)
 
 #undef _NSIG
